Userland/Libraries/LibCore/ConfigFile.cpp

```diff
@@ -160,6 +160,20 @@ bool ConfigFile::read_bool_entry(String const& group, String const& key, bool de
     return value == "1" || value.equals_ignoring_case("true"sv);
 }
 
+Color ConfigFile::read_color_entry(String const& group, String const& key, Color default_value) const
```

Comment:
I don't think LibCore should have any dependencies on LibGfx. This feels like the wrong place. Maybe something like `Color::to_string` or `Color::from_string` if we don't already have it?

Reply:
There was already write_color_entry() in ConfigFile, so this function is not adding any dependencies that were not already there.

Reply:
There is already Color::to/from_string)() though, so I guess we can use that instead. Though I feels a little weird to remove the old config API in this PR too

Reply:
Oh I didn't realize that. Still, seems weird to me that there's any LibGfx related stuff in LibCore.